styles/styles.css

```diff
@@ -131,6 +139,7 @@ body {
   padding: 20px 10px;
   width: 80%;
   max-width: 600px;
+  transform: translateY(-8vh);
```

Comment:
@Mandla-tech The 'transform: translate' is pushing the modal to high up on the screen and is potentially causing issues with the buttons which we saw on the call when screen sharing. Please can you remove this?